scripts/trace-acceptor.sh

```diff
@@ -2,9 +2,9 @@
 
```

Comment:
I don't think changes to this file need to be included in this PR.

Reply:
it is just to make sure this and `shelley-testnet.sh` are using both `cabal`.
makes it much easier to launch the demo.

Reply:
restored